alembic/versions/466569e48bf4_added_unique_to_uuid.py

```diff
@@ -0,0 +1,23 @@
+"""added unique to uuid
+
+Revision ID: 466569e48bf4
+Revises: 0ca2cc448074
+Create Date: 2018-12-03 09:40:48.931574
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '466569e48bf4'
+down_revision = '0ca2cc448074'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_unique_constraint(op.f('uq_users_uuid'), 'users', ['uuid'])
```

Comment:
Hmm this shouldn't work in SQLite due to the incomplete ALTER TABLE support no?

Reply:
Yep. Totally right. Sqlite yells at me for this. I think we need to get #55 in before we do any more migrations because we have some broken things already in the codebase.